packages/cyberstorm/src/components/PackageList/PackageList.tsx

```diff
@@ -22,81 +24,133 @@ interface Props {
 
 const PER_PAGE = 20;
 
-/**
- * Fetches packages based on props and shows them as a list.
- *
- * TODO: add loading indicator over the PackageCard section. Doing so
- *       properly might require rethinking the structure of things, i.e.
- *       adding another wrapping component.
- */
-export function PackageList(props: Props) {
+export function PackageList(props: PackageListProps) {
   const { categories, deprecated, listingType, nsfw, searchQuery, section } =
     props;
 
   const [order, setOrder] = useState(PackageOrderOptions.Updated);
   const [page, setPage] = useState(1);
+  const [packageCount, setPackageCount] = useState<number | null>(null);
 
-  const deferredCategories = useDeferredValue(categories);
```

Comment:
1. FYI, the deferred values were used originally because there were errors that the slow fetching of packages blocked the rest of the UI without them. I didn't see the errors now, so I guess something had changed and they might not be needed anymore.
2. Getting rid off them to get the Suspense fallback show again might be ok, but from what I understood that's frowned upon and the best practice would be to use the fallbacks only when loading the initial data. After that updates should be indicated in a more subtle manner. See https://react.dev/reference/react/Suspense#showing-stale-content-while-fresh-content-is-loading and https://react.dev/reference/react/useDeferredValue for more info. Again, if we truly want to fallback to the skeleton components, I guess it's okay to do this from the technical perspective, as long as it doesn't block the rest of the UI.

Reply:
We don't have it in the designs, but I guess covering the stale data with some mist and a loading icon, would make more sense.

Reply:
Decided to just add some opacity